angular15-vue3/vue-remote/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,46 @@
+<template>
```

Comment:
Why we need separate duplicated component in order to use it as a custom element?

Reply:
to prevent styling issues for custom elements described https://github.com/vuejs/core/issues/4662